packages/core/src/interfaces/ubi/beneficiary.ts

```diff
@@ -86,3 +86,12 @@ export interface BeneficiaryCreationAttributes {
     tx: string;
     txAt: Date;
 }
+
+export interface BeneficiarySubgraph {
+    claims: number;
+    community: {
+        id: string;
+    };
+    lastClaimAt: number;
+    preLastClaimAt: number;
+}
```

Comment:
maybe I would keep this on a separate file, where, later, might have all TheGraph entities types.

Reply:
right, I change to `src/subgraph/interfaces` to be with everything related to subgraph